lib/services/protocol/happn_4.js

```diff
@@ -111,6 +111,17 @@ class ProtocolHappn4 extends BaseHappnProtocol {
     )
       response.data = this.__formatReturnItems(response.initialValues, local);
 
+    if (message.action === 'set' && message.options && message.options.noDataResponse)
```

Comment:
What of older servers like fieldpop?

Reply:
Again - the data would be returned, not removed from the output message, not sure how to make something forward-compatible except top release it to the cloud?